### DISCUSSION TITLE
Is it's possible to get the results distances normalized to range of 0-1 or 0-100

### DISCUSSION THREAD

**Discussion by sandraplato1234** [milvus-io/milvus#4712] [Q&A and General discussion]
Hi,
I'm using the IVFFLAT IP index and I got as search result's distances that are more than 100 it's it possible to get them normalized ? 
Thanks in advance

**Comment by gujun720** [comment]
Did you normalize the vectors before they are inserted into Milvus?

**Reply by sandraplato1234** [reply]
Hi,
yes- faiss_normalize_L2
When I insert the vectors not normalized had the same issue distances up to 130
After insert them with normalization i got up to 13
